service/src/main/java/bio/terra/cbas/dao/mappers/GithubMethodSourceDetailsMapper.java

```diff
@@ -0,0 +1,20 @@
+package bio.terra.cbas.dao.mappers;
+
+import bio.terra.cbas.models.GithubMethodDetails;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.UUID;
+import org.springframework.jdbc.core.RowMapper;
+
+public class GithubMethodSourceDetailsMapper implements RowMapper<GithubMethodDetails> {
```

Comment:
Looking at the convention and to keep it consistent, can this also be renamed to ` GithubMethodDetailsMapper`?